app/models/concerns/alegre_v2.rb

```diff
@@ -383,14 +383,19 @@ def get_required_keys(project_media, field)
       }
     end
 
+    def get_parsed_cached_data_for_key(key)
+      value = redis.get(key)
+      Hash[YAML.load(value).collect{|kk,vv| [kk.to_i, vv]}] if value
```

Comment:
@DGaffney would be very useful if you could add here a comment with an example of this conversion